components/ws-daemon/pkg/content/hooks.go

```diff
@@ -29,12 +29,12 @@ func workspaceLifecycleHooks(cfg Config, kubernetesNamespace string, workspaceEx
 			hookSetupWorkspaceLocation,
 			startIWS, // workspacekit is waiting for starting IWS, so it needs to start as soon as possible.
 			hookSetupRemoteStorage(cfg),
-			hookInstallQuota(xfs),
+			hookInstallQuota(xfs, false),
 		},
 		session.WorkspaceReady: {
 			startIWS,
 			hookSetupRemoteStorage(cfg),
-			hookInstallQuota(xfs),
+			hookInstallQuota(xfs, true),
```

Comment:
Question: if we restore from the backup is over 30G and we set the hard limit after the workspace is ready, is it okay to set the hard limit?
Or no, once we restore from the backup, it should no longer than 30G?

Reply:
@jenting Good question! It is assumed that backups exceeding 30 GB are not possible. This is because of xfs limitations when the workspace is terminated. Please let me know if there is a situation where it exceeds 30GB. I would like to try it.